privacyidea/api/lib/postpolicy.py

```diff
@@ -775,3 +775,63 @@ def construct_radius_response(request, response):
         return make_response('', return_code)
     else:
         return response
+
+
+def mangle_challenge_response(request, response):
+    """
+    This policy decorator is used in the AUTH scope to
+    decorate the /validate/check endpoint.
+    It can modify the contents of the response "detail"->"message"
+    to allow a better readibilty for a challenge response text.
+
+    :param request:
+    :param response:
+    :return:
+    """
+    content = json.loads(response.data)
```

Comment:
Uh, just noticed because of Travis: In case of ``/validate/radiuscheck``, the response body is empty, so no JSON object can be decoded: https://travis-ci.org/privacyidea/privacyidea/jobs/493770918#L2185
Should we check for ``if response.data:``? Or just do nothing in case of a ValueError?

Reply:
What do you mean - just do nothing?

We could also do an early exit, if the endpoint is not ``/validate/check``?

Reply:
By "do nothing", I meant we could catch the ValueError and do an early exit:
```python
try:
    content = json.loads(response.data)
except ValueError:
    return response
```
But an early exit in case the endpoint is not ``/validate/check`` also sounds good!

Reply:
done in a94cc884